cluster/manifests/flannel/daemonset.yaml

```diff
@@ -37,20 +36,10 @@ spec:
             memory: 50Mi
       containers:
       - name: delayed-install-cni
-        image: container-registry.zalando.net/teapot/flannel-awaiter:master-9
+        image: container-registry.zalando.net/teapot/flannel-awaiter:master-10
```

Comment:
Can we make this an `initContainer` instead? That seems to better capture the use case and results in one less container that's running.

Reply:
It probably needs the local flannel running before the `await` can ever succeed, right?

Reply:
Yes, we discussed making it an init container but then the parallel running is necessary, so kept it as it is.